api/src/db/models/competition_models.py

```diff
@@ -139,6 +144,10 @@ class Application(ApiSchemaTable, TimestampMixin):
         "ApplicationForm", uselist=True, back_populates="application", cascade="all, delete-orphan"
     )
 
+    users: Mapped[list["ApplicationUser"]] = relationship(
```

Comment:
I'd suggest we call this `application_users` so it's sightly more specific as these don't refer to the actual user records.